subxt/src/events.rs

```diff
@@ -157,7 +157,6 @@ impl<T: Runtime + System> EventsDecoder<T> {
             let mut event_data = Vec::<u8>::new();
             let mut event_errors = Vec::<RuntimeError>::new();
             let result = self.decode_raw_bytes(
-                module.name(),
```

Comment:
Need to re-apply some of our changes

Reply:
I've done this
https://github.com/Phala-Network/phala-blockchain/pull/553/commits/d0a65bfa15a94d3b3af0bcce37b368ac1acbdd35

I checked https://github.com/paritytech/subxt/pull/302/files?authenticity_token=VNjJrTCHNqs6JXze5OT%2FG6DfmHPq0jDobV5bQQmmwR0wFL%2BxMZsEeuY9pX3COvPh7Ymstnl3Oq7sC%2BUy83NpOA%3D%3D&file-filters%5B%5D=.rs#diff-59e0fecb904d610f27811d416aa3f26cc0d1bfed220ac69f8e18147f9bd68fd0 doesn't change events.rs so it's safe to revert the file